proposed price with decimal support in histogram

xtine opened the pull request:
Addresses both #214 and #215

theresaanna commented on hourglass_site/static/hourglass_site/js/index.js:
```diff
@@ -57,6 +57,20 @@
       return true;
   }
 
+  function getSigFigs(num) {
```
@xtine Would you mind commenting this function a bit? Its not immediately clear how it does what it does, particularly given the reg exes.

theresaanna commented on hourglass_site/static/hourglass_site/js/index.js:
```diff
@@ -1298,6 +1321,12 @@ if(getUrlParameterByName('proposed-price').length) {
   });
 
+ $('.two-decimal-places').keyup(function(){
+    if(!(/^\d+(\.{0,1}\d{0,2})?$/.test(this.value))){
```
@xtine Can we also comment this regex?

xtine commented:
Thanks for the catch @theresaanna. Cleaned up the code and removed an overly complex function for what could have been done in a cleaner one line. Also added comment about the two decimal place regex functionality.
